### PR TITLE
Apply black style to Bio.Phylo.

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #2008 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

I think this is fine, but is a long change it would be good to have it reviewed.

### CODE REVIEW

File: Bio/Phylo/Applications/_Phyml.py [hunk 1]
```diff
@@ -33,39 +33,40 @@ class PhymlCommandline(AbstractCommandline):
...
+                """Use a minimum parsimony starting tree.
 
                     This option is taken into account when the '-u' option is absent
                     and when tree topology modifications are to be done.
-                    """
-                    ),
-
-            _Option(["-b", "--bootstrap", "bootstrap"],
-                    r"""Number of bootstrap replicates, if value is > 0.
+                    """,
```
Comment (peterjc) [inline Bio/Phylo/Applications/_Phyml.py]: This looks wrong - the opening and closing triple quotes are not aligned, and changing the opening line only probably changes the interpretation of the indentation of the text?

Comment (svalqui) [reply Bio/Phylo/Applications/_Phyml.py]: Yes, you are right, looks odd, I'll align them, it just needs to be done manually.

**Comment by peterjc** [comment]
I've opened #2369 for the triple-quote issue, which is separate from the black work - but that did drawn attention to it.

If you want to fix the command line parameter help text indentation as part of this pull request, fine - but otherwise I suggest you leave the triple-quote strings exactly as black puts them, and we'll deal with the indentation later?

File: Bio/Phylo/Applications/_Phyml.py (hunk 1 above)
Comment (peterjc) [reply Bio/Phylo/Applications/_Phyml.py]: I was wrong about how Python interprets these strings - but these help text strings were and still are not optimal. See #2369 and main comment.

**Comment by svalqui** [comment]
Ok, I'm leaving it as black style; triple quotes alignment for later.

**Pull request merged**